docs/interfaces/rest.md

```diff
@@ -0,0 +1,77 @@
+# REST Interface
+
+## Prerequisites
+
+* [Query Lifecycle](./query-lifecycle.md)
+* [Application CLI](./cli.md)
+
+## Synopsis
+
+This document describes how to create a REST interface for an SDK application. A separate document for creating module REST Routes can be found [here](#../module-interfaces.md#rest).
+
+- [Application REST Interface](#application-rest-interface)
+- [Request Types](#request-types)
+- [REST Server](#rest-server)
+- [Registering Routes](#registering-routes)
+
+## Application REST Interface
+
+Building the REST Interface for an application involves creating a REST server to route requests and output responses. The SDK has its own REST Server type used for LCDs (light-client daemons). It has a `ServeCommand` that takes in an application's `codec` and `RegisterRoutes` function, starts up a new SDK REST Server, and registers routes using function provided from the application. To enable this command, it should be added as a subcommand of the root command `RootCmd` in the `main()` function of the CLI interface.
+
+Users can use the application CLI to start a new LCD, a local server through which they can securely interact with the application without downloading the entire state. The command entered by users would look something like this:
+
+```bash
+appcli rest-server --chain-id <chainID> --trust-node
+```
+
+## Request Types
```

Comment:
From there, needs to be reworked. The information is not bad, but it does not fit with the synopsis, which states "This document describes how to create a REST interface for an SDK application.". This doc should explain the basics of a REST Server, and explain how devs can create one for their app. Please don't forget to add snippets/links/examples to contextualize